course/theme.json

```diff
@@ -824,7 +824,7 @@
 						}
 					}
 				},
-				"css": "& .grade { font-family: var(--wp--preset--font-family--system) }"
+				"css": "& .grade { font-family: var(--wp--preset--font-family--system) } & .sensei-lms-question__answer-feedback--incorrect{ --color: #f2ddbe; }"
```

Comment:
Curious if the default theme needs the more specific `& .sensei-lms-question-block__header .grade` selector or no?